sse2neon.h

```diff
@@ -8873,13 +8873,48 @@ static inline int _sse2neon_sido_negative(int res, int lb, int imm8, int bound)
 }
 
 #define SSE2NEON_MIN(x, y) (x) < (y) ? (x) : (y)
-#define SSE2NEON_GET_LENGTH_OR_BOUND(la, lb, bound) \
-    int tmp1 = la ^ (la >> 31);                     \
-    la = tmp1 - (la >> 31);                         \
-    int tmp2 = lb ^ (lb >> 31);                     \
-    lb = tmp2 - (lb >> 31);                         \
-    la = SSE2NEON_MIN(la, bound);                   \
-    lb = SSE2NEON_MIN(lb, bound);
+
+#define SSE2NEON_CMPESTRX_GET_UPPER_BOUND \
+    const int bound = (imm8 & 0x01) ? 8 : 16
```

Comment:
Is `upper` better than `bound`?

Reply:
Macro `SSE2NEON_GET_LENGTH_OR_BOUND` might be refined for the sake of consistency.

Reply:
> Is `upper` better than `bound`?

I think the word "bound" represents the boundary of string length. Hence, I consider using "bound" is suitable.

Reply:
> I think the word "bound" represents the boundary of string length. Hence, I consider using "bound" is suitable.

How about changing the form of function-like macro?
```c
#define SSE2NEON_CMPESTRX_SET_UPPER(var, imm) \
    const int var = (imm & 0x1) ? 8 : 16
```

Then, we can use it below:
```c
    SSE2NEON_CMPESTRX_SET_UPPER(bound);
```

Reply:
> How about changing the form of function-like macro?

It looks better than current implementation. I will adopt this in next commit.